app/addons/components/components/copy.js

```diff
@@ -0,0 +1,71 @@
+// Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Clipboard from 'clipboard';
+
+let clipboard;
+
+export const initializeClipboard = (cb) => {
+  clipboard = new Clipboard('.copy');
+  if (cb != null) {
```

Comment:
I don't really like this `cb` check. This basically means this will fail silently. Rather make sure that the react component always checks for a function via prop types that is passed into here.